File: new_modun_4/module_4/CaseStudy/src/main/java/com/example/casestudy/customer/model/Customer.java
package com.example.casestudy.customer.model;


import javax.persistence.*;

@Entity
@Table(name = "customer")
public class Customer {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int idCustomer;
    private String nameCustomer;
    private String dataOfBirth;
    private String gender;
    private String idCard;
    private String phoneNumer;
    private String email;
    private String address;
}

File: new_modun_5/customer/baiThi/src/app/model/customer.ts
import {CustomerType} from "./customerType";

export interface Customer {
  id?:number;
  name?:string;
  dateOfBirth?: string;
  idCard?: string;
  phone?: string;
  email?: string;
  address?: string;
  gender?:string;
  customerType?: CustomerType;
}
